### PR TITLE
CNV-43385: Added networking architecture diagram

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.15+

Issue: [CNV-43385](https://issues.redhat.com//browse/CNV-43385)

Link to docs preview: https://79229--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-networking-overview.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: virt/vm_networking/virt-networking-overview.adoc [hunk 1]
```diff
@@ -8,6 +8,8 @@ toc::[]
 
 {VirtProductName} provides advanced networking functionality by using custom resources and plugins. Virtual machines (VMs) are integrated with {product-title} networking and its ecosystem.
 
+image::318_OpenShift_Virtualization_Networking_0423.png[{VirtProductName} Networking architecture]
+
```
Comment (phoracek) [inline virt/vm_networking/virt-networking-overview.adoc:12]: Suggested text (to explain the basic concepts, but to make sure people understand this is not the only solution we have to offer):

Illustration of a typical network setup of OpenShift Virtualization. Other options are available.

1. Pods and VMs are running on the same network infrastructure, allowing users to easily connect their containerized and virtualized workloads
2. VMs can be connected to the default pod network and to any number of secondary networks
3. The default pod network provides connectivity between all its members, service abstraction, it supports IP management, micro segmentation, and more
4. Multus is a component allowing users to connect to secondary networks
5. The default pod network is overlay-based, tunneled through the underlying machine network
6. The machine network can be defined over a selected set of NICs
7. Secondary VM networks are typically bridged directly to a physical network, with or without VLAN encapsulation
8. Secondary VM networks can be defined on dedicated set of NICs (in the picture) or they can use the machine network

**Review by phoracek — APPROVED** [review]
LGTM, thanks!

**Review by lpettyjo — COMMENTED** [review]
LGTM!

**Review by yossisegev — CHANGES_REQUESTED** [review]
From this illustration it looks like the default pod networks goes through the multus CNI, but it doesn't.
@phoracek Please keep me honest here so I won't cause unnecessary confusion.

**Comment by phoracek** [comment]
Technically it does - Multus is the main CNI, delegating to the default Pod network CNI and secondary CNIs. Although many people take Multus (or Multus networks) as a synonym of secondary networks. I would keep the picture as is.

**Comment by yossisegev** [comment]
> Technically it does - Multus is the main CNI, delegating to the default Pod network CNI and secondary CNIs. Although many people take Multus (or Multus networks) as a synonym of secondary networks. I would keep the picture as is.

Very well then. So the change LGTM.

**Review by yossisegev — APPROVED** [review]

**Pull request merged**